setup.py

```diff
@@ -13,6 +15,26 @@ def convert(filename, fmt):
         with io.open(filename, encoding='utf-8') as fd:
             return fd.read()
 
+def create_mo_files():
+    data_files = []
+    localedir = 'cmsplugin_cascade/locale'
+    po_dirs = [localedir + '/' + l + '/LC_MESSAGES/'
+               for l in next(os.walk(localedir))[1]]
+    for d in po_dirs:
+        mo_files = []
+        po_files = [f
+                    for f in next(os.walk(d))[2]
+                    if os.path.splitext(f)[1] == '.po']
+        for po_file in po_files:
+            filename, extension = os.path.splitext(po_file)
+            mo_file = filename + '.mo'
```

Comment:
use the functions provided by Python to handle file extensions.

Reply:
You talk about the python function mimetypes ?   (.mo, text/x-modelica )